packages/graphiql/README.md

```diff
@@ -6,11 +6,109 @@
+
+I would suggest a search for "graphiql <my runtime>" such as "graphiql express", "graphiql absinthe", etc to learn a potentially simpler route to setup for your environment. There are many npm packages, ruby gems, java utilities for deploying graphiql.
+
+Here are some example searches:
+
+- https://www.npmjs.com/search?q=graphiql - ~117 hits
+- https://pypi.org/search/?q=graphiql - ~33 hits
+- https://search.maven.org/search?q=graphiql - ~15 hits
+- https://rubygems.org/search?utf8=%E2%9C%93&query=graphiql - ~6 hits
+- https://godoc.org/?q=graphiql - ~12 hits
+- https://packagist.org/?query=%22graphiql%22 - ~5 hits
+- https://crates.io/search?q=graphiql - ~2 hits
+
+though this doesnt include runtimes or libraries where `graphiql` is used isn't referenced in the package registry search entry
+
+### CDN Bundle
+
+Don't forget to include the CSS file on the page! If you're using `npm` or `yarn`, you can find it in `node_modules/graphiql/graphiql.css`, or you can download it from the [releases page](https://github.com/graphql/graphiql/releases).
+
+For an example of setting up a GraphiQL, check out the [example](../examples/graphiql-cdn/) in this repository which also includes a few useful features highlighting GraphiQL's API.
+
+The most minimal way to set up GraphiQL is a single index.html file:
+
+```html
+<html>
+  <head>
+    <title>Simple GraphiQL Example</title>
+    <link href="https://unpkg.com/graphiql/graphiql.min.css" rel="stylesheet" />
+  </head>
+  <body style="margin: 0;">
+    <div id="graphiql" style="height: 100vh;"></div>
+
+    <script crossorigin src="https://unpkg.com/react/umd/react.min.js"></script>
+    <script
+      crossorigin
+      src="https://unpkg.com/react-dom/umd/react-dom.min.js"
+    /></script>
+    <script crossorigin src="https://unpkg.com/graphiql/graphiql.min.js" />
+
+    <script>
+      function graphQLFetcher(graphQLParams) {
+        return fetch('https://my/graphql', {
+          method: 'post',
+          headers: { 'Content-Type': 'application/json' },
+          body: JSON.stringify(graphQLParams),
+        }).then(function(response) {
+          return response.json();
+        });
+      }
+      ReactDOM.render(
+        React.createElement(GraphiQL, { fetcher: graphQLFetcher }),
+        document.getElementById('graphiql'),
+      );
+    </script>
+  </body>
+</html>
+```
+
+**Notes**:
+
+- the inlined styles are important for ensuring GraphiQL is visible and fills the canvas.
+- using `React.createElement` directly is belaborous, so follow the webpack instructions below for more highly customized implementation
+
+### Webpack/Bundler
+
+**Note**: If you are having webpack issues or questions about webpack, make sure you've cross-referenced your webpack configuration with our own [webpack example](../examples/graphiql-webpack) first. We confirm that this builds on every travis CI run, and that it works end to end with every publish.
```

Comment:
```suggestion
**Note**: If you are having webpack issues or questions about webpack, make sure you've cross-referenced your webpack configuration with our own [webpack example](../examples/graphiql-webpack) first. We now have tests in CI that ensure this always builds, and we ensure it works end-to-end with every publish.
```

How do we ensure it works with every publish - manually?

Reply:
in CI

Reply:
Ah; so should it be `with every merge` then, or do we have a specific CI test that only runs on publish?

Reply:
not sure whats best. would like shorter builds. the demo project webpack build only adds about 22s. so adding a few Cypress assertions would be easy, but for now i can just confirm by going to <deploy-preview-path.netlify>.com/webpack before merging. would be good to automate at least a basic test, but the cypress suite should be focused on the minified build

Reply:
currently, we just do `yarn build-demo` in CI to confirm that webpack can import and resolve the GraphiQL package, and that it can perform a webpack build without all the assets that are present in `packages/graphiql`. otherwise, build-demo will not exit 0